core/src/i18n/en.yaml

```
@@ -1244,7 +1244,7 @@ jobs:
     containers-readonly-in-edit: After a Job is created, the containers are read-only.
     labels:
       args: Args
-      command: Command
+      command: Commands
```

It's a single command, consisting of multiple parts.
```suggestion
      command: Command
```